cmd/apply.go

```diff
@@ -20,13 +20,12 @@
 package cmd
 
 import (
+	"context"
 	"errors"
 	"fmt"
-	"io/ioutil"
-	"context"
-
 	"github.com/cloudspannerecosystem/wrench/pkg/spanner"
 	"github.com/spf13/cobra"
+	"os"
```

Comment:
Please separate standard libraries and third party libraries by empty line and put this package to standard libraries group.

Reply:
Thank you for your review! I ran goimports and fixed the order!